code/modules/hydroponics/grown/ambrosia.dm

```diff
@@ -80,7 +80,7 @@
 	product = /obj/item/weapon/reagent_containers/food/snacks/grown/ambrosia/cruciatus
 	potency = 10
 	mutatelist = list()
-	reagents_add = list("thc" = 0.15, "kelotane" = 0.15, "bicaridine" = 0.1, "bath_salts" = 0.20, "nutriment" = 0.05)
+	reagents_add = list("thc" = 0.15, "kelotane" = 0.15, "bicaridine" = 0.1, "bath_salts" = 0.20, "plantmatter" = 0.05)
```

Comment:
The gaia, mimana, and holy melons I intentional kept as nutriment, otherwise, it's virtually impossible to acquire; anything of a more ethereal/magical/supernatural nature I let have nutriment, since it's a universal nutrition provider.